packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -3080,11 +3080,6 @@ class _TextSelectionGestureDetectorState extends State<TextSelectionGestureDetec
     }
   }
 
-  @override
```

Comment:
Curious: Why did the unnecessary_overrides lint not catch this one? https://dart-lang.github.io/linter/lints/unnecessary_overrides.html

Reply:
Yes I do mention it in my explanations above... That's weird